explorer/explorerroutes.go

```diff
@@ -205,8 +205,27 @@ func (exp *explorerUI) AddressPage(w http.ResponseWriter, r *http.Request) {
 		addrHist, balance, errH := exp.explorerSource.AddressHistory(
 			address, limitN, offsetAddrOuts)
 		if errH != nil {
-			log.Errorf("Unable to get address %s history: %v", address, errH)
-			exp.ErrorPage(w, "Something went wrong...", "could not find that address's history", false)
+			log.Infof("Unable to get address %s history: %v", address, errH)
+			addrData := exp.blockData.GetExplorerAddress(address, limitN, offsetAddrOuts)
```

Comment:
If this actually finds addresses, but the PG query did not, there needs to be a stern `log.Error` message.

Reply:
No, the one above should still be `Infof`.  I meant after the `if addrData == nil { ... }` block below, add a new Error message (just dcrdata console, not the web page) because this should not happen. It means the PG DB has malfunctioned